javascript/ql/experimental/adaptivethreatmodeling/lib/experimental/adaptivethreatmodeling/EndpointFeatures.qll

```diff
@@ -558,6 +567,13 @@ class Callee_AccessPath extends EndpointFeature, TCallee_AccessPath {
  * "Containment" is syntactic, and currently means that the endpoint is an argument to the call, or that the endpoint is a (nested) property value of an argument.
  *
  * This feature is intended as a superior version of the `ArgumentIndexFeature`.
+ *
+ * Examples:
+ * ```
+ * foo(endpoint); // -> 0
+ * foo({ bar: endpoint }); // -> 0.bar
+ * foo(x, { bar: { baz: endpoint } }); // -> 1.bar.baz
```

Comment:
Why do we need the argument index here? We already have it as a separate feature `Input_ArgumentIndex`. Why not just have this be `Input_ArgumentAccessPathFromCallee`?

Reply:
For historical reasons I suppose by now.
This was the initial feature by a static analysis engineer, the feature value is how a static analysis would treat the endpoint. 
I suppose your reasoning for dropping this is that it is fine to let the model figure out the relationship between the argument index and the access path by itself?

Reply:
Yes, as well as a couple more considerations: https://github.com/github/ml-ql-adaptive-threat-modeling/issues/1828#issuecomment-1083525582

Reply:
Right. That was probably the comment that motivated the split in the first place.

I have removed the redundant feature in 6f64db1 below.

Reply:
Thank you! I hadn't noticed there's already a feature `Input_AccessPathFromCallee`